RubeesEat/Model/DB/DbPersonRepository.cs

```diff
@@ -65,28 +84,65 @@ public void Add(Person person)
     
     public Person GetCurrentUser()
     {
-        return GetAll()[^1];
+        var httpContext = _httpContextAccessor.HttpContext;
+        if (httpContext == null)
+            throw new NullReferenceException("HttpContext is null");
+        var loginName = httpContext.User.FindFirst("name")?.Value;
+        var nickName = httpContext.User.FindFirst("nickname")?.Value;
+        var splitNickName = nickName.Split(".");
+        var firstName = splitNickName[0];
+        var lastName = splitNickName[1];
+        if (loginName == null)
+            throw new NullReferenceException("User is inactive");
```

Comment:
Yes, Patrick pointed this out to us. NullReferenceExceptions shouldn't be thrown on purpose because the compiler throws them, so if you throw them on purpose it just makes understanding where the error message comes from more difficult.

Reply:
This isn't updated as of the time I got this in re-review so I'll keep the convo open until it's fixed :)

Reply:
Sorry, I somehow overlooked it and thought it was showing a change that was already made, instead of the code suggestion from you